example/ck_tile/35_batched_transpose/transpose_2d.cpp

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2025, Advanced Micro Devices, Inc. All rights reserved.
+
+#include <cassert>
+#include <cstdlib>
+#include <iostream>
+#include <numeric>
+#include <time.h>
+#include <unordered_set>
+#include <vector>
+
+#include "batched_transpose.hpp"
+
+auto create_args(int argc, char* argv[])
+{
+    ck_tile::ArgParser arg_parser;
+    arg_parser.insert("v", "1", "whether do CPU validation or not")
+        .insert("N", "2", "input batch size. ")
+        .insert("H", "16", "input height size.")
+        .insert("W", "16", "input width size. ")
+        .insert("batch_stride", "0", "Batch stride, 0 means H * W.")
+        .insert("pr", "fp16", "input data type. fp16/fp32 (representing 8/16/32 bit data)")
+        .insert("seed", "-1", "seed to be used, -1 means random every time");
+
+    bool result = arg_parser.parse(argc, argv);
+    return std::make_tuple(result, arg_parser);
+}
+
+template <typename Type>
+bool run_transpose(ck_tile::ArgParser args)
+{
+    int validate     = args.get_int("v");
+    std::string prec = args.get_str("pr");
+    int N            = args.get_int("N");
+    int H            = args.get_int("H");
+    int W            = args.get_int("W");
+    int seed         = args.get_int("seed");
+    int batch_stride = args.get_int("batch_stride");
```

Comment:
it's no necessary to specify batch_stride in 2d example, since the batch size is always 1